init.lua

```diff
@@ -7,8 +7,7 @@ signs_lib = {}
 
 signs_lib.path = minetest.get_modpath(minetest.get_current_modname())
 
-local S = dofile(signs_lib.path .. "/intllib.lua")
-signs_lib.gettext = S
+local S = minetest.get_translator(minetest.get_current_modname())
```

Comment:
`S` is not used in `init.lua` and could be removed. However I suggest following old method with something like:
```lua
signs_lib.S = minetest.get_translator(.......)
```
Then in `api.lua` and other places that actually use the translator:
```lua
local S = signs_lib.S
```